osis-core/src/main/java/com/scality/osis/service/impl/ScalityOsisService.java

```diff
@@ -89,7 +91,38 @@ public OsisTenant createTenant(OsisTenant osisTenant) {
 
     @Override
     public PageOfTenants queryTenants(long offset, long limit, String filter) {
-        throw new NotImplementedException();
+        if(filter.contains(CD_TENANT_ID_PREFIX)) {
+            try {//cd_tenant_id==e7ecb16e-f6b7-4d34-ad4e-5da5d2c83009
```

Comment:
left over comment?

Reply:
Good catch I will remove this

Reply:
fixed this in b55a95980e673a114c719f9405fa10b8e6199d10